static/js/zamboni/devhub.js

```diff
@@ -885,8 +885,8 @@ function initCatFields(delegate) {
             $("input", $main).attr("checked", false).attr("disabled", false);
```

Comment:
I see there's quite a few bad usages of `.attr` instead of `.prop` that we'll need to fix in another PR ;)